images/makeJs_mac.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/zsh
```

Comment:
Is this needed because MacOS doesn't have bash anymore?

This script doesn't use any obvious bashisms, so `#!/bin/sh` might be sufficient.

Reply:
You are correct, will update